kubernetes/linux/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN /usr/bin/apt-get update && /usr/bin/apt-get install -y libc-bin wget openssl
 COPY setup.sh main.sh defaultpromenvvariables defaultpromenvvariables-rs defaultpromenvvariables-sidecar mdsd.xml envmdsd logrotate.conf $tmpdir/

```

Comment:
FYI.. we should clean up Dockerfile since this not being used.

Reply:
discussed this will be addressed in May release task